builtin/k8s/releaser.go

```diff
@@ -320,11 +320,13 @@ func (r *Releaser) resourceServiceCreate(
 		nodeclient := clientSet.CoreV1().Nodes()
 		nodes, err := nodeclient.List(ctx, metav1.ListOptions{})
 		if err != nil {
-			return err
+			// Rather than fail the whole release, report the error and then complete.
+			step.Status(terminal.StatusError)
+			step.Update("Cannot determine release URL for nodeport service due to failure to list nodes: %s", err)
```

Comment:
Maybe this message should just be on its own with a `c.ui.Output` and a `terminal.WithWarningStyle()`? Otherwise I think this updates whatever the last stepgroup line was, and could potentially be updated over if we add any other stepgroup output later. It could also be its own step if we do the `sg.Add()` and close it with done right away.